<launch>
  <node name="localizer_node" pkg="localizer" type="localizer_node" output="screen"/>
</launch>